### DISCUSSION TITLE
Can we support migrations on GitHub website?

### DISCUSSION THREAD

**Discussion by Goooler** [github/gh-actions-importer#62] [Feedback] [answered]
If I haven't installed CLI, just import from other CI configs on the website, it would be much easier to use.

**Comment by ethanis** [comment, marked as answer]
Hi @Goooler! There is not a GUI for using GitHub Actions Importer built into github.com. 

However, one option that may be of interest is the [actions/importer-issue-ops](https://github.com/actions/importer-issue-ops) template repo. You can use this to set up a way to run Actions Importer commands through IssueOps without needing to install anything on your local machine.

**Reply by Goooler** [reply]
Good to know!
